glastopf/modules/handlers/emulators/rfi.py

```diff
@@ -21,12 +21,55 @@
 import re
 import logging
 import ssl
+import socket
+import requests
+from urlparse import urlparse
+from socket import inet_aton
+from struct import unpack
+from requests.utils import requote_uri
+
+
 
 import glastopf.sandbox.sandbox as sandbox
 from glastopf.modules.handlers import base_emulator
 
 logger = logging.getLogger(__name__)
 
+def check_ssrf(url):
```

Comment:
what are you actually checking here?

Reply:
Can you find a better name for this function?

Reply:
maybe you can give me a better name.